src/views/protected/MyAccount/MyAccountPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
+import {
+    Button,
+    FormControl,
+    FormHelperText,
+    Grid,
+    Input,
+    InputLabel,
+    makeStyles,
+    Typography,
+    withStyles,
+} from '@material-ui/core'
+import * as Types from '../../../constants/Types'
+import AppBar from '../../../components/AppBar'
+import Firebase, { FirebaseContext } from '../../../components/Firebase'
+import CHALKLogoGIF from '../../../assets/images/CHALKLogoGIF.gif'
+import { connect } from 'react-redux'
+import { Help } from '@material-ui/icons'
+
+
+const useStyles = makeStyles(_ => ({
+    root: {
+        flexGrow: 1,
+        height: '100vh',
+        flexDirection: 'column',
+        overflowX: 'hidden',
+        overflowY: 'auto',
+    },
+    formControl: {
+        minWidth: 300,
+    },
+    formContainer: {
+        marginTop: '2.3em',
+    },
+    comparisonText: {
+        paddingLeft: '1em',
+        lineHeight: '0.8em',
+        fontFamily: 'Arimo',
+    },
+    container: {
+        marginLeft: '1em',
+    },
+    select: {
+        width: '15em',
+    },
+}))
+
+const StyledFormControl = withStyles(() => ({
+    root: {
+        marginBottom: '1.3rem',
+    },
+}))(FormControl)
+
+const save = (name: string,
```

Comment:
@thompchr can we end stuff on the backend before deploying to actually save the changes to the password and user info?